hack/local-up-cluster.sh

```diff
@@ -129,6 +129,10 @@ KUBE_CONTROLLERS="${KUBE_CONTROLLERS:-"*"}"
 # Audit policy
 AUDIT_POLICY_FILE=${AUDIT_POLICY_FILE:-""}
 
+# The timeout setting When waiting node ready with 'kubectl get nodes'
+NODE_READY_TIMEOUT_SECOND=${NODE_READY_TIMEOUT_SECOND:-150}
```

Comment:
please set this to `60` like it was before. you can use the `NODE_READY_TIMEOUT_SECOND` to 150 in your environment.

Reply:
But the point of this PR is to increase the default timeout. I assume other developers will have similar experience when running local cluster of containerd.

Is there a good reason to keep 60s?